ExeterTeam25/group-software-project
# Interactive map implemented

To test:
- make sure folium is installed (pip install folium)
- map shows up on the home page with markers on it
- the map should zoom in and out 
- the map should snap back to the centre when dragged too far

Details:
- added the map functionality into the app
- currently, you cannot obtain the lat and long when clicking a location on the map
